### PR TITLE
feat(lang): emulate language on firefox

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: test/emulation.spec.js [hunk 1]
```diff
@@ -308,7 +308,7 @@ module.exports.describe = function({testRunner, expect, playwright, headless, FF
       expect(await page.evaluate(() => navigator.language)).toBe('fr-CH');
       await context.close();
     });
-    it('should format number', async({browser, server}) => {
+    it.fail(FFOX)('should format number', async({browser, server}) => {
```
Comment (aslushnikov) [inline test/emulation.spec.js]: why doesn't this work?

**Review by aslushnikov — APPROVED** [review]

File: src/firefox/ffBrowser.ts [hunk 2]
```diff
@@ -168,6 +168,11 @@ export class FFBrowserContext extends BrowserContextBase {
   }
 
   async _initialize() {
+    if (this._options.locale) {
+      if (!this._options.extraHTTPHeaders)
+        this._options.extraHTTPHeaders = {};
+      this._options.extraHTTPHeaders['Accept-Language'] = this._options.locale;
```
Comment (dgozman) [inline src/firefox/ffBrowser.ts]: I think we should leave `options.extraHTTPHeaders` as is and instead update FFBrowserContext.setExtraHTTPHeaders.

File: test/emulation.spec.js (hunk 1 above)
Comment (pavelfeldman) [reply test/emulation.spec.js]: We need to patch builtins w/ locale data.

File: src/firefox/ffBrowser.ts (hunk 2 above)
Comment (pavelfeldman) [reply src/firefox/ffBrowser.ts]: Done

**Pull request merged**